src/seedu/addressbook/AddressBook.java

```diff
@@ -1146,12 +1151,13 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+    	String regex = '^' + prefix;
+        return fullString.replaceFirst(regex, "");
```

Comment:
What if the prefix is not at the start of fullString?  Have to check using startsWith first before using replaceFirst.